### PR TITLE
Sanitize user name upon account creation

### PR DESCRIPTION
**Pull request by abought**
Immediate short-term fix to scrub malicious HTML from user names upon registration. Will send further details of fix to @lbanner , @GaryKriebel , or @JeffSpies by email and discuss in-person as needed.

### CODE REVIEW

**Comment by lbanner** [comment]
:+1: @sloria let's prioritize this one. TY

File: framework/auth/views.py [hunk 1]
```diff
@@ -246,12 +247,14 @@ def register_user(**kwargs):
             http.BAD_REQUEST,
             data=dict(message_long='Email addresses must match.')
         )
-    # TODO: Sanitize fields
     try:
+        full_name = request.json['fullName']
```
Comment (HarryRybacki) [inline framework/auth/views.py]: Any reason not to combine these two lines?

Comment (abought) [inline framework/auth/views.py]: No special reason for or against- I just find it more readable not to chain statements in this case.

**Comment by sloria** [comment]
Will merge this in now since it's a priority. @abought Can you add a regression test in another PR?

**Comment by abought** [comment]
Will do, @sloria. Thanks for the reminder.

**Comment by icereval** [comment]
@abought please resubmit this against the master branch

**Pull request merged**